dediprog.c

```diff
@@ -363,6 +363,10 @@ static int dediprog_set_spi_speed(unsigned int spispeed_idx)
 static int prepare_rw_cmd(
 		struct flashctx *const flash, uint8_t *data_packet, unsigned int count,
```

Comment:
I wonder why count is not just a u16, which gives us a bit more enforcement at compile time ...